ballerina-release-notes-0.990.0.md

```diff
@@ -239,9 +297,15 @@ io:println(var2);
 }
 	var x => io:println(“Matched to Default Pattern”); //all values will match
 }
-
 ```
 
+### Dynamically narrowing type
+We talked about two types of relationships between a value and a type in this section: looks like and belongs to. With the introduction of these two relationships, one might want to transform a value that looks like a type into a value that belongs to that type. We’ve introduced two built-in methods to do that.
```

Comment:
Can we reword this? The way this is worded here makes me think that "looks like and belongs to" is the name of a section in the release notes, and that's not the case.